Libraries/ReactIOS/YellowBox.js

```diff
@@ -82,9 +84,29 @@ function updateWarningMap(format, ...args): void {
     ...args.slice(argCount).map(stringifySafe),
   ].join(' ');
 
-  const count = _warningMap.has(warning) ? _warningMap.get(warning) : 0;
-  _warningMap.set(warning, count + 1);
+  var warningInfo = _warningMap.get(warning);
```

Comment:
The eslint-bot doesn't complain in existing code, right? If so, we can enable the let/const rule